Imagine/Cache/Resolver/WebPathResolver.php

```diff
@@ -63,7 +63,7 @@ public function resolve($path, $filter)
      */
     public function isStored($path, $filter)
     {
-        return $this->filesystem->exists($this->getFilePath($path, $filter));
+        return is_file($this->getFilePath($path, $filter));
```

Comment:
I do not have a link handy but afaik this isn't relevant in terms of performance.